sale_product_set/wizard/product_set_add.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from openerp import models, fields, api, _
+from odoo import models, fields, api, _
 import openerp.addons.decimal_precision as dp
 
 
```

Comment:
Can you also fix the two field's strings here by removing _()

Plus the typo on _descritpion _description